tests/js/web/Tooltips/TooltipListenerTest.js

```diff
@@ -0,0 +1,134 @@
+import $ from 'jquery';
+import TooltipListener from '../../../../js/Tooltips/TooltipListener'
+
+describe('TooltipListener', () => {
+    let $html;
+    let $body;
+    let tooltipListener;
+    let $button;
+    let tippyStub;
+    let hideAllStub;
+    let keyDownEvent = $.Event('keydown');
+
+    beforeEach(() => {
+        $html = $('html');
+        $body = $('body');
```

Comment:
can we use fake html and body nodes here? Like just regular divs? This sort of stuff will break the browser version of the test suite (which are admittedly broken already!)